exercises/02-robot-syntax.md

```diff
@@ -14,17 +14,16 @@ Check that welcome page text is visible
 
 </details>
 
-If you now run command `robot robot/login.robot` you should get something like this:
+
+If you now run the command `robot robot/login.robot`, you should get something like this:
 
 `[ ERROR ] Parsing 'robot/login.robot' failed: File has no tests or tasks.`
 
 or
 
 `[ ERROR ] Suite 'Login' contains no tests or tasks.`
 
-The reason is that this is not in Robot syntax.
-
-Check out Robot Framework user guide: [Creating test data](http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#creating-test-data).
```

Comment:
We definitely want to keep links to user guide in the material. "Half" of the point of the training is to familiarize the participants that if they need more information they can look at the user guide. The point is not to read the user guide in the training, so the wording can be a little different, but the link should definitely stay in.